components/style/properties/longhand/box.mako.rs

```diff
@@ -368,47 +368,42 @@ ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
                    animatable="False">
     use self::computed_value::{StartEnd, TransitionTimingFunction};
 
-    use euclid::point::Point2D;
+    use euclid::point::{Point2D, TypedPoint2D};
+    use std::marker::PhantomData;
 
     pub use self::computed_value::SingleComputedValue as SingleSpecifiedValue;
     pub use self::computed_value::T as SpecifiedValue;
 
-    static EASE: TransitionTimingFunction = TransitionTimingFunction::CubicBezier(Point2D {
-        x: 0.25,
-        y: 0.1,
-    }, Point2D {
-        x: 0.25,
-        y: 1.0,
-    });
-    static LINEAR: TransitionTimingFunction = TransitionTimingFunction::CubicBezier(Point2D {
-        x: 0.0,
-        y: 0.0,
-    }, Point2D {
-        x: 1.0,
-        y: 1.0,
-    });
-    static EASE_IN: TransitionTimingFunction = TransitionTimingFunction::CubicBezier(Point2D {
-        x: 0.42,
-        y: 0.0,
-    }, Point2D {
-        x: 1.0,
-        y: 1.0,
-    });
-    static EASE_OUT: TransitionTimingFunction = TransitionTimingFunction::CubicBezier(Point2D {
-        x: 0.0,
-        y: 0.0,
-    }, Point2D {
-        x: 0.58,
-        y: 1.0,
-    });
-    static EASE_IN_OUT: TransitionTimingFunction =
-        TransitionTimingFunction::CubicBezier(Point2D {
-            x: 0.42,
-            y: 0.0,
-        }, Point2D {
-            x: 0.58,
-            y: 1.0,
-        });
+    #[inline(always)]
```

Comment:
Oh, also, sorry for not suggesting this before, but could you add here a `FIXME: Use static + const fn when available` or similar?